src/libclipper/src/rpc_service.cpp

```diff
@@ -28,16 +29,19 @@ namespace clipper {
 namespace rpc {
 
 constexpr int INITIAL_REPLICA_ID_SIZE = 100;
+constexpr int CONTACTING_GAP_TOLORANCE = 30;
+constexpr double CONTAINER_EXISTENCE_CHECK_FREQUENCY = 0.2;
```

Comment:
What does `.2` mean in terms of time? Let's make this a long called `CONTAINER_ACTIVITY_CHECK_FREQUENCY_MILLIS` and define it to be `10000` (10 seconds, in milliseconds).

Reply:
Well, these constants I defined was in seconds, and the CONTAINER_EXISTENCE_CHECK_FREQUENCY is the time gap between the two continuous container existence check, so wouldn't 10 seconds be too long.